buildkitd/dockerd-wrapper.sh

```diff
@@ -120,6 +120,13 @@ start_dockerd() {
     data_root=$(TMPDIR="$EARTHLY_DOCKERD_DATA_ROOT/" mktemp -d)
     echo "Starting dockerd with data root $data_root"
 
+    if uname -a | grep microsoft-standard-WSL >/dev/null; then
+        if iptables --version | grep nf_tables >/dev/null; then
+            echo "WARNING: WSL and iptables-nft may not work; attempting to switch to iptables-legacy"
+            ln -sf "/sbin/iptables-legacy" /sbin/iptables
+        fi
+    fi
+
```

Comment:
@alexcb In latest versions of debian its also like that..
In most of my scripts I have..
```
    DO github.com/earthly/lib+INSTALL_DIND
    # Bit hacky hack to fix docker on new systems
    RUN update-alternatives --set iptables /usr/sbin/iptables-legacy
    DO +TEST_WITH_DOCKER --cmd="some"
```